.github/workflows/system-test.yml

```
@@ -0,0 +1,50 @@
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+  workflow_dispatch: { }
+
+jobs:
+  toml-cases:
+    runs-on: ubuntu-latest
+    outputs:
+      test-names: ${{ steps.find-tests.outputs.test-names }}
```

actionlint (1.7.4)

property "find-tests" is not defined in object type {}

(expression)